modules/recursion-schemes/arrow-recursion/src/test/kotlin/arrow.recursion/laws/RecursiveLaws.kt

```diff
@@ -0,0 +1,14 @@
+package arrow.recursion.laws
+
+import arrow.recursion.typeclasses.Corecursive
+import arrow.recursion.typeclasses.Recursive
+import arrow.test.laws.Law
+import io.kotlintest.matchers.shouldEqual
+
+object RecursiveLaws {
```

Comment:
We have all laws in the `arrow-laws` module which should depend on all `arrow` libraries, if you prefer to have them there!

Reply:
Makes sense, I'll move them there.

Reply:
`forAll(intGen)` can become `forFew`, the rest you can move to the Generators file :D